install-base.sh

```diff
@@ -120,10 +121,17 @@ fi
 # Add the Cylc wrapper scripts
 dos2unix -n /vagrant/usr/local/bin/cylc /usr/local/bin/cylc
 cd /usr/local/bin
+ln -sf cylc isodatetime
 ln -sf cylc gcylc
 # Configure additional copyable environment variables
 mkdir -p /opt/metomi-site/conf
 dos2unix -n /vagrant/opt/metomi-site/conf/global.rc /opt/metomi-site/conf/global.rc
+mkdir -p /opt/metomi-site/etc/cylc/flow/8
+dos2unix -n /vagrant/opt/metomi-site/etc/cylc/flow/8/global.cylc /opt/metomi-site/etc/cylc/flow/8/global.cylc
+# Insecure workaround for browser permissions error
+# See https://stackoverflow.com/questions/70753768/jupyter-notebook-access-to-the-file-was-denied
+mkdir -p /opt/metomi-site/etc/cylc/uiserver
+dos2unix -n /vagrant/opt/metomi-site/etc/cylc/uiserver/jupyter_config.py /opt/metomi-site/etc/cylc/uiserver/jupyter_config.py
```

Comment:
This workaround exposes the auth token in the process list so would not be appropriate on a shared system.